Examples/Python/src/TrackFitting.cpp

```diff
@@ -66,6 +66,14 @@ void addTrackFitting(Context& ctx) {
         py::arg("reverseFilteringMomThreshold"),
         py::arg("freeToBoundCorrection"), py::arg("level"));
 
+    py::class_<MeasurementCalibrator, std::shared_ptr<MeasurementCalibrator>>(
+        mex, "MeasurementCalibrator");
+
+    mex.def("makePassThroughCalibrator",
```

Comment:
I know this is merged already, but I there a reason not to just expose the inheritance relationship and then have Python just construct the object directly instead of the factory?

Reply:
No real reason I guess, I just don't know what I'm doing on the python side ;-) putting this on my list